source/materials/MaterialsList.cc

```diff
@@ -478,6 +478,8 @@ G4Material* MaterialsList::TPB()
 
 G4Material* MaterialsList::ITO()
 {
+  // Data from http://www.mit.edu/~6.777/matprops/ito.htm
```

Comment:
This appears to be a random reference, and the link may eventually stop working. Instead, you could include the DOI of the papers they cite.

Reply:
It's weird. The link works for me without any problem.
In it you can find several paper cites. The one associated to the density is https://doi.org/10.1116/1.1371326.
I can change the current one  by this one if you prefer.

Reply:
Sorry, I didn't mean the link doesn't work. I meant that it is someone's personal website, and thus the link may stop working any time. I'd rather include the link to the paper, yes.

Reply:
Just because the link works now, doesn't mean it won't be gone by next week!

Reply:
Done